components/src/LiveCode/index.tsx

```diff
@@ -0,0 +1,25 @@
+/** @jsx jsx */
+import { jsx } from 'theme-ui'
+import Default from './liveCodeScope'
```

Comment:
```suggestion
import defaultScope from './liveCodeScope'
```
Remember to give variables/imports a meaningful name, and also that you should only begin their names with capital letters if they're components, interfaces, or types :sparkles: